dev-lang/zig/zig-0.9.1-r1.ebuild

```diff
@@ -24,6 +24,7 @@ RESTRICT="!test? ( test )"
 PATCHES=(
 	"${FILESDIR}/${P}-fix-single-threaded.patch"
 	"${FILESDIR}/${P}-fix-riscv.patch"
+	"${FILESDIR}/${P}-fix-bad-hostname-segfault.patch"
```

Comment:
Yep, to answer your earlier question: revbump needed as it fixes a runtime problem. It's not fixing e.g. a defective test (as in the test itself being based on an incorrect premise), it's fixing an issue a test exposed.

See https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html.

Reply:
> Yep, to answer your earlier question: revbump needed as it fixes a runtime problem. It's not fixing e.g. a defective test (as in the test itself being based on an incorrect premise), it's fixing an issue a test exposed.
> 
> See https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html.

Thank you!